trieste/acquisition/rule.py

```diff
@@ -1068,3 +1081,118 @@ def state_func(
             return state_, points
 
         return state_func
+
+
+class BatchHypervolumeSharpeRatioIndicator(
+    AcquisitionRule[TensorType, SearchSpaceType, ProbabilisticModelType]
+):
+    def __init__(
+        self, batch_size: int = 5, population_size: int = 500, filter_threshold: float = 0.4
+    ):
+        if pymoo is None:
```

Comment:
(remember to add a docstring explaining the rule and parameters: half the people will try and figure out how to use this from the tutorial, the other half from here)